deploy/microshift-bootc/config-svc/app.py

```
@@ -0,0 +1,2302 @@
+            }
+        }
+        
+        # Write CR to temporary file
+        with tempfile.NamedTemporaryFile(mode='w', suffix='.yaml', delete=False) as f:
+            yaml_content = json_to_yaml(cr)
+            f.write(yaml_content)
+            temp_file = f.name
+        
+        try:
+            # Apply using oc with explicit kubeconfig
+            result = subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)